webpack/webpack.config.js

```diff
@@ -11,11 +11,13 @@ function requireAll(array) {
 
 module.exports = {
   entry: "./src/js/index.js",
+  devtool: "inline-source-map",
   output: {
     filename: "[name].js",
     path: path.resolve(__dirname, "../dist")
   },
   resolve: {
+    extensions: [".ts", ".tsx", ".js"],
```

Comment:
Do we want to use the `tsx` suffix? We don't use `.jsx`, so I would opt-in for a no here

Reply:
Could not make the `tsc` understand jsx syntax on a `ts` file. So if we don't support `tsx`  that means we do not support using types on a react component.

That is also the same way it is used on ui-kit.

Reply:
Let's see if we can teach that somehow to `tsc`. I really dislike the idea of having ts and tsx files. Let's timebox it to an hour and if we don't succeed I am fine with that

Reply:
Actually, my timebox for typescript is already over. If you have you or someone else have some time to do it, that would be awesome. But I will be picking it up once I have graphql in the data layer.

Reply:
@DanielMSchmidt TS is strict about the file type and if I'm not wrong it only accepts ts or tsx.

I also suggest using ts/tsx to TS files and js/jsx for JS files because even though they are both JS, TS is a language (e.g even GH support TS/TSX as a different language than JS).

Reply:
I would recommend us to bring this one to sync for discussion next week. I am putting the topic on the sync for the case we cant solve until there. Maybe we should also research if it is even possible to have ts file only (so far I could not).